worker/src/test/java/gov/cms/ab2d/worker/processor/ContractProcessorInvalidPatientTest.java

```diff
@@ -0,0 +1,159 @@
+        map.put("3", ContractBeneficiaries.PatientDTO.builder().patientId("3").dateRangesUnderContract(dates).build());
+        tracker.addPatientsByContract(cb);
+        ReflectionTestUtils.setField(cut, "cancellationCheckFrequency", 20);
+        ReflectionTestUtils.setField(patientClaimsProcessor, "startDate", "01/01/2020");
+    }
+
+    @Test
+    void testInvalidBenes() throws IOException {
+        Bundle b1 = BundleUtils.createBundle(createBundleEntry("1"));
+        Bundle b2 = BundleUtils.createBundle(createBundleEntry("2"));
+        Bundle b4 = BundleUtils.createBundle(createBundleEntry("4"));
+        when(bfdClient.requestEOBFromServer(eq("1"), any())).thenReturn(b1);
+        when(bfdClient.requestEOBFromServer(eq("2"), any())).thenReturn(b2);
+        when(bfdClient.requestEOBFromServer(eq("3"), any())).thenReturn(b4);
+        List<JobOutput> outputs = cut.process(tmpDirFolder.toPath(), contractData);
+        assertNotNull(outputs);
+        assertEquals(2, outputs.size());
+        String fileName1 = contractId + "_0001.ndjson";
+        String fileName2 = contractId + "_0002.ndjson";
+        String output1 = outputs.get(0).getFilePath();
+        String output2 = outputs.get(1).getFilePath();
+        assertTrue(output1.equalsIgnoreCase(fileName1) || output1.equalsIgnoreCase(fileName2));
+        assertTrue(output2.equalsIgnoreCase(fileName1) || output2.equalsIgnoreCase(fileName2));
+        String actual1 = Files.readString(Path.of(tmpDirFolder.getAbsolutePath() + "/" + output1));
+        String actual2 = Files.readString(Path.of(tmpDirFolder.getAbsolutePath() + "/" + output2));
+        assertTrue(actual1.contains("Patient/1") || actual1.contains("Patient/2"));
+        assertTrue(actual2.contains("Patient/1") || actual2.contains("Patient/2"));
+    }
+
+    @Test
+    void testWriteErrors() throws IOException {
+        Job job = new Job();
+        job.setJobUuid(jobId);
+        String val = "Hello World";
+        StreamHelper helper = new TextStreamHelperImpl(
+                tmpDirFolder.toPath(), contractId, 2000, 10, eventLogger, job);
+        ((ContractProcessorImpl) cut).writeExceptionToContractErrorFile(
+                helper, val, new RuntimeException("Exception"));
+        String result = Files.readString(Path.of(tmpDirFolder.getAbsolutePath() + "/" + contractId + "_error.ndjson"));
+        assertEquals(val, result);
+    }
+
+    @Test
+    void testWriteNullErrors() throws IOException {
+        Job job = new Job();
+        job.setJobUuid(jobId);
+        StreamHelper helper = new TextStreamHelperImpl(
+                tmpDirFolder.toPath(), contractId, 2000, 10, eventLogger, job);
+        ((ContractProcessorImpl) cut).writeExceptionToContractErrorFile(
+                helper, null, new RuntimeException("Exception"));
+        assertThrows(NoSuchFileException.class, () -> Files.readString(Path.of(tmpDirFolder.getAbsolutePath() + "/" + contractId + "_error.ndjson")));
```

Comment:
Use  File.separator or some other environmentally defined path seperation.  Forward slash does not compare will on my box.